doc/tutorials/calib3d/camera_multiview_calibration/multiview_calibration.markdown

```diff
@@ -0,0 +1,226 @@
+* Rotation / Translation vectors of extrinsics with respect to camera 0. Number of vectors NUM_CAMERAS - 1, for the first camera rotation / translation vector is zero.
+* Intrinsic matrix for each camera.
+* Distortion coefficients for each camera.
+* Rotation / Translation vector of each frame pattern with respect to camera 0.
+* Matrix of reprojection errors of size NUM_CAMERAS x NUM_FRAMES
+* Output pairs used for initial estimation of extrinsics. NUM_CAMERAS - 1.
+
+Pseudocode:
+----
+```{r, eval = FALSE}
+def mutiviewCalibration (pattern_points, image_points, visibility_matrix):
+  for cam_i = 1,…,NUMBER_CAMERAS:
+    if CALIBRATE_INTRINSICS:
+      K_i, distortion_i, rvecs_i, tvecs_i = calibrateCamera(pattern_points, image_points[cam_i])
+    else:
+      rvecs_i, tvecs_i = solvePnP(pattern_points, image_points[cam_i], K_i, distortion_i)
+  Select best rvecs, tvecs based on reprojection errors.
+  Process data:
+    pattern_img_area[cam_i][frame] = area(convexHull(image_points[cam_i][frame]
+    angle_to_board[cam_i][frame] = arccos(pattern_normal_frame * optical_axis_cam_i)
+    angle_cam_to_cam[cam_i][cam_j] = arccos(optical_axis_cam_i * optical_axis_cam_j)
+  graph = maximumSpanningTree(visibility_mat, pattern_img_area, angle_to_board, angle_cam_to_cam)
+  camera_pairs = bread_first_search(graph, root_camera=0)
+  for pair in camera_pairs:
+    # find relative rotation, translation from camera i to j
+    R_ij, t_ij = stereoCalibrate(pattern_points, image_points[i], image_points[j])
+  R*, t* = optimizeLevenbergMarquardt(R, t, pattern_points, image_points, K, distortion)
+```
+
+Python samples:
+----
+To demonstrate functionally of the proposed method, the corresponding sample file is created in Python.
+Its arguments are either a path to a JSON file already containing all image and pattern points, together with camera information; or files containing image names, and camera information passed through a command line.
+If the arguments are files containing images, the function automatically does detection (the pattern type has to be specified, e.g., checkerboard).
+Apart from estimated extrinsics / intrinsics, the python sample provides a comprehensive visualization.
+
+Firstly, the sample shows positions of cameras, checkerboard (of a random frame), and pairs of cameras connected by black lines explicitly demonstrate tuples used in the initial stage of stereo calibration.
+
+If images are not known, then a simple plot with arrows (from given point to the back-projected one) visualizing errors are shown. The color of arrows highlights the error values. Additionally, the title reports mean error on this frame, and its accuracy among other frames used in calibration.
+
+The following test instances were synthetically generated (see `opencv/apps/python-calibration-generator/calibration_generator.py`):
+
+<p align="center">
+  <img src="images/1a.png" width="400" />
+  <img src="images/1b.png" width="286" />
+</p>
```

Comment:
I build documentation locally and doxygen does not insert images: "Firefox cannot file file /home/alexander/opencv-next-build/doc/doxygen/html/images/1a.png"

Reply:
fixed